getting-started.html

```diff
@@ -0,0 +1,149 @@
+    </div>
+
+    <div class="start-step-container">
+        <div class="start-step">
+            <div class="step-text">
+                <h2>Download and Install Bisq</h2>
+                <p>You can download and install Bisq easily <a href="/downloads">from this website</a> or <a href="https://github.com/bisq-network/bisq/releases/latest" target="_blank">from GitHub</a>.</p>
+                <p>It's also a good idea to verify your installer file to be extra sure it hasn't been meddled with. See the videos here for directions. More details are in <a href="https://bisq.wiki/Downloading_and_installing" target="_blank">this wiki article</a>.</p>
+                <p>If you'd like to build Bisq from source, <a href="https://github.com/bisq-network/bisq/blob/master/docs/build.md" target="_blank">here are directions</a>. Bisq is free/libre open-source software that <a href="https://docs.bisq.network/contributor-checklist.html" target="_blank">you can contribute to</a>.</p>
+            </div>
+            <div class="step-feature">
+                <div class="feature-selector"><a href="#" id="select-windows-install-video" class="install-step-item active">Windows</a><a href="#" id="select-linux-install-video" class="install-step-item">macOS or Linux</a></div>
+                <div class="responsive-youtube-container install-step-item active" id="windows-install-video">
+                    <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/XABzwXw6X0A" frameborder="0" allow="accelerometer; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+                </div>
+                <div class="responsive-youtube-container install-step-item" id="linux-install-video">
+                    <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/dTfM4AsxNHY" frameborder="0" allow="accelerometer; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+                </div>
+            </div>
+        </div>
+
+        <div class="start-step">
+            <div class="step-text">
+                <h2>Set Up and Tour</h2>
+                <p>With Bisq, you're in total control of your funds and your data. This means you retain unparalleled sovereignty, but it also means no one can help you if you lose something important—so it's critical that you do proper backups before using Bisq to trade.</p>
+                <p>We've also prepared a short tour of the Bisq interface so you can make the most of it.</p>
+                <p>At the very least, please be 100% sure to <a href="https://bisq.wiki/Backing_up_your_wallet" target="_blank">write down your seed words</a> and <a href="https://bisq.wiki/Backing_up_a_data_directory" target="_blank">back up your data directory</a>.</p>
```

Comment:
"At the very least, please be 100% sure to backup .." => "Follow these simple steps (+link to wiki) to backup.."

The first version sounds like its kind of optional and we're kind of asking them to maybe consider doing it, at the very least maybe somehow think about it a bit.

Second version: hey go here and do it :) More imperative, people will see it as less optional, probably more will do it.